climada/hazard/test/test_base_xarray.py

```diff
@@ -0,0 +1,353 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2022 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Test xarray reading capabilities of Hazard base class.
+"""
+
+import os
+import unittest
+import datetime as dt
+import numpy as np
+from scipy.sparse import csr_matrix
+
+import xarray as xr
+
+from climada.hazard.base import Hazard
+
+THIS_DIR = os.path.dirname(__file__)
+
+
+class ReadDefaultNetCDF(unittest.TestCase):
+    """Test reading a NetCDF file where the coordinates to read match the dimensions"""
+
+    def setUp(self):
+        """Write a simple NetCDF file to read"""
+        self.netcdf_path = os.path.join(THIS_DIR, "default.nc")
```

Comment:
Using `pathlib` instead of `os` is recommended.
```suggestion
        self.netcdf_path = Path.cwd() / "default.nc"
```
The you can remove the ugly 
`THIS_DIR = os.path.dirname(__file__)`
and import
`from pathlib import Path`

Reply:
Yes, that looks better! Maybe we should even use [`tempfile`](https://docs.python.org/3/library/tempfile.html) to create a temporary file/directory for that? 🤔

Reply:
Could also be nice. I haven't used it often yet myself though. Feel free to choose what you prefer!